posts/2021-02-19-configure-trustedHeaderOrigin-21002.adoc

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "Configure trustedHeaderOrigin and trustedSensitiveHeaderOrigin to be configured with a IP segment with Open Liberty 21.0.0.2"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jkaub-pomykala
+seo-title: Configure trustedHeaderOrigin and trustedSensitiveHeaderOrigin to be configured with a IP segment with Open Liberty 21.0.0.2 - OpenLiberty.io
+seo-description: With Open Liberty 21.0.0.2 you can now configure `trustedHeaderOrigin` and `trustedSensitiveHeaderOrigin` using IP address wildcards and hostnames. Also included a number of significant bug fixes.
+blog_description: With Open Liberty 21.0.0.2 you can now configure `trustedHeaderOrigin` and `trustedSensitiveHeaderOrigin` using IP address wildcards and hostnames. Also included a number of significant bug fixes.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Configure trustedHeaderOrigin and trustedSensitiveHeaderOrigin to be configured with a IP segment with Open Liberty 21.0.0.2
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+// tag::intro[]
+
+With Open Liberty 21.0.0.2 you can now configure `trustedHeaderOrigin` and `trustedSensitiveHeaderOrigin` using IP address wildcards and hostnames.
+
+In link:{url-about}[Open Liberty] 21.0.0.2:
+
+* <<trustedHeader, trustedHeaderOrigin Configuration Improvements>>
+* <<bugs, Significant bugs fixed in this release>>
+
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3ARELEASE_VERSION+label%3A%22release+bug%22[21.0.0.2].
+// end::intro[]
+
+
+// tag::run[]
+[#run]
+
+== Run your apps using 21.0.0.2
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
```

Comment:
Don't change this yet but if it looks good on the beta blog then we might consider moving this section to the bottom??